src/cbapi/psc/threathunter/models.py

```diff
@@ -174,16 +174,16 @@ class Event(UnrefreshableModel):
     """Events can be queried for via ``CbThreatHunterAPI.select``
     or though an already selected process with ``Process.events()``.
     """
-    urlobject = '/threathunter/search/v1/orgs/{}/events/_search'
-    validation_url = '/threathunter/search/v1/orgs/{}/events/search_validation'
+    urlobject = '/api/investigate/v2/orgs/{}/events/{}/_search'
```

Comment:
I am not sure the injection of the process_guid into the URL is properly handled in the _search function for class Query. This will cause the request to fail if not specified

Reply:
Nevermind, I missed the url component in _search